app/controllers/application_controller.rb

```diff
@@ -5,5 +5,8 @@ class ApplicationController < ActionController::Base
 
   def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    rescue ActiveRecord::RecordNotFound
```

Comment:
@stovermc I'm gonna end up not using this, because Sorcery handles this by default, but the idea is if a user WAS logged in, and we reset the database (I.E. ran `rake db:test:prepare`) the session would have a user ID, so the session would be valid, but the user wouldn't show up and we'd see "ActiveRecord::RecordNotFound"

So this line just says "if that error is thrown, `rescue` it by clearing the session"

The next line is session.clear, which reset the session.